packages/beacon-node/test/e2e/eth1/jsonRpcHttpClient.test.ts

```diff
@@ -1,343 +1,342 @@
 import "mocha";
 import crypto from "node:crypto";
 import http from "node:http";
-import chai, {expect} from "chai";
-import chaiAsPromised from "chai-as-promised";
+import {expect} from "chai";
 import {JsonRpcHttpClient} from "../../../src/eth1/provider/jsonRpcHttpClient.js";
 import {getGoerliRpcUrl} from "../../testParams.js";
 import {IRpcPayload} from "../../../src/eth1/interface.js";
 
-chai.use(chaiAsPromised);
-
-describe("eth1 / jsonRpcHttpClient", function () {
-  this.timeout("10 seconds");
-
-  const port = 36421;
-  const noMethodError = {code: -32601, message: "Method not found"};
-  const notInSpecError = "JSON RPC Error not in spec";
-  const randomHex = crypto.randomBytes(32).toString("hex");
-
-  const testCases: {
-    id: string;
-    url?: string;
-    payload?: IRpcPayload;
-    requestListener?: http.RequestListener;
-    abort?: true;
-    timeout?: number;
-    error: any;
-  }[] = [
-    // // NOTE: This DNS query is very expensive, all cache miss. So it can timeout the tests and cause false positives
-    // {
-    //   id: "Bad domain",
-    //   url: `https://${randomHex}.com`,
-    //   error: "getaddrinfo ENOTFOUND",
-    // },
-    {
-      id: "Bad subdomain",
-      // Use random bytes to ensure no collisions
-      url: `https://${randomHex}.infura.io`,
-      error: "getaddrinfo ENOTFOUND",
-    },
-    {
-      id: "Bad port",
-      url: `http://localhost:${port + 1}`,
-      requestListener: (req, res) => res.end(),
-      error: "connect ECONNREFUSED",
-    },
-    {
-      id: "Not a JSON RPC endpoint",
-      requestListener: (req, res) => {
-        res.setHeader("Content-Type", "text/html");
-        res.end("<html></html>");
+describe("jsonRpcHttpClient", () => {
+  describe("eth1 / jsonRpcHttpClient", function () {
```

Comment:
Is it really necessary to nest? Would prefer to keep the diff small even tho it has two top level describes :pray:

Reply:
It's not recommended to have multiple top level describe. Here is the analogy for it. 

> Multiple describe blocks is often a sign that a test should be broken down to multiple files. One of the possible problems if having multiple suites is that, if you are, for example, going through tests one by one and focusing them (using describe.only), you might not notice describe block(s) that are down there at the bottom of a file which may lead to tests being unintentionally skipped.

Find more details here. 
https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/max-top-level-suites.md

Reply:
Right I see their point but I disagree. Again that's a purely organizational decision. Would prefer to have that in a different PR and discuss there, else this PR may be left often longer than necessary.